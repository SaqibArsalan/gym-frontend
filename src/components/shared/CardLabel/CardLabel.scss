@import 'assets/scss/Variables.scss';
@import 'assets/scss/Extends.scss';

.data-table-label {
	flex: 1 1 100%;
	padding: toRem(16);
	font-size: toRem(14);
	color: $gray;
	text-align: left;
}
