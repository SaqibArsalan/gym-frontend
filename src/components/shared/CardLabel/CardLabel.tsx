import React from 'react';

import { Typography } from '@mui/material';

import './CardLabel.scss';

interface ICardLabel {
	label: string;
}

const CardLabel = (props: ICardLabel) => {
	const { label } = props;
	return (
		<Typography className='data-table-label' id='title' component='div'>
			{label}
		</Typography>
	);
};

export default CardLabel;
