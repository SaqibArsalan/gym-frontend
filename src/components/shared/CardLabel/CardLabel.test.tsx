import React from 'react';
import "@testing-library/react/dont-cleanup-after-each";
import {cleanup, render, RenderResult } from '@testing-library/react';
import '@testing-library/jest-dom'; 
import CardLabel from './CardLabel';

let renderWrapper: RenderResult;


describe("CardLabel tests", () => {
    beforeAll(() => {
        renderWrapper = render(<CardLabel label='test' />);
    });
    afterAll(() => {
        cleanup();
    });
    test("default renders successfully", () => {
        expect(renderWrapper.container).not.toBeNull();
        expect(renderWrapper.container).toMatchSnapshot();
    });
})