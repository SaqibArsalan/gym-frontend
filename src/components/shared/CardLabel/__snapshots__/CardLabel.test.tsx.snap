// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardLabel tests default renders successfully 1`] = `
<div>
  <div
    class="MuiTypography-root MuiTypography-body1 data-table-label css-ahj2mt-MuiTypography-root"
    id="title"
  >
    test
  </div>
</div>
`;
